holoviews/plotting/bokeh/callbacks.py

```diff
@@ -92,19 +104,41 @@ class Callback(object):
     attributes = {}
 
     js_callback = """
+        function unique_events(events) {{
```

Comment:
As far as I can tell this is a self-contained function that could be a utility. Would be good to have a docstring i.e a comment line saying that it does and the same for ``process_events`` below. At some point we will probably want the concept of Javascript 'utilities' that you can just include where needed.

That said, I just noticed ``comm_state`` is referenced in this function, can it not be made into an explicit function argument?